### PR TITLE
Added optimal meatballs notebook

### PR DESCRIPTION
**Pull request by mthelm85**
Per @fonsp request, I'm submitting the optimal meatballs notebook. I wasn't sure which category to place it in (plot or math)

### CODE REVIEW

File: src/plot/Optimal_Meatballs.jl [hunk 1]
```diff
@@ -0,0 +1,1248 @@
...
+# v0.19.36
+
+#> [frontmatter]
+#> license_url = "https://unlicense.org/"
+#> title = "Optimal Meatballs"
+#> date = "2024-02-08"
+#> tags = ["circle-packing", "Plots.jl", "PlutoUI.jl"]
+#> description = "Learn about circle packing by optimizing the size of meatballs!  🍝"
+#> license = "Unlicense"
+#>
```
Comment (lukavdplas) [inline src/plot/Optimal_Meatballs.jl:11]: Adding a cover image would be nice, [maybe this image?](https://commons.wikimedia.org/wiki/File:Polpettes_et_linguini_Pasta_%C3%A9_Pizza_(Lyon)_mars_2023_(2).jpg). It has a CC0 licence, so you can freely reuse it here.
```suggestion
#> image = "https://upload.wikimedia.org/wikipedia/commons/thumb/f/fb/Polpettes_et_linguini_Pasta_%C3%A9_Pizza_%28Lyon%29_mars_2023_%282%29.jpg/540px-Polpettes_et_linguini_Pasta_%C3%A9_Pizza_%28Lyon%29_mars_2023_%282%29.jpg"
#>
```

**Review by lukavdplas — COMMENTED** [review]
This is a really neat idea! 😄

**Comment by lukavdplas** [comment]
> I wasn't sure which category to place it in (plot or math)

Hmm, I think plot may be more appropriate? What do you think, @fonsp ?

File: src/plot/Optimal_Meatballs.jl (hunk 1 above)
Comment (mthelm85) [reply src/plot/Optimal_Meatballs.jl:11]: Yes! I forgot to add a cover image.

**Comment by fonsp** [comment]
Thanks @mthelm85! I remember your github name, you were one of the first Pluto users, right? Very happy to see you're still enjoying Pluto!!

For the category – maybe "Puzzles & Games"? But we can always change it later

File: src/plot/Optimal_Meatballs.jl [hunk 2]
```diff
@@ -0,0 +1,1249 @@
...
+#> title = "Optimal Meatballs"
+#> date = "2024-02-08"
+#> tags = ["circle-packing", "Plots.jl", "PlutoUI.jl"]
+#> description = "Learn about circle packing by optimizing the size of meatballs!  🍝"
+#> license = "Unlicense"
+#> image = "https://upload.wikimedia.org/wikipedia/commons/thumb/f/fb/Polpettes_et_linguini_Pasta_%C3%A9_Pizza_%28Lyon%29_mars_2023_%282%29.jpg/540px-Polpettes_et_linguini_Pasta_%C3%A9_Pizza_%28Lyon%29_mars_2023_%282%29.jpg"
+#>
+#>     [[frontmatter.author]]
+#>     name = "Matt Helm"
+#>     url = "https://avatars.githubusercontent.com/u/30706758?v=4"
```
Comment (fonsp) [inline src/plot/Optimal_Meatballs.jl:15]:
```suggestion
#>     url = "https://github.com/mthelm85"
```

**Comment by mthelm85** [comment]
@fonsp Yes, I believe I was an early adopter : ) I made a tiny contribution (with your help) back in 2020: https://github.com/fonsp/Pluto.jl/pull/247

I also created the unregistered [PlutoDataTable.jl package](https://github.com/mthelm85/PlutoDataTable.jl). 

What you all have done with Pluto.jl/PlutoUI.jl has been fantastic - congratulations and I hope to see it continue to grow!!

**Pull request closed without merging**